src/app/containers/ArticleMain/index.jsx

```diff
@@ -18,10 +19,13 @@ const componentsToRender = {
 };
 
 const ArticleMain = ({ articleData }) => {
+  const { pageType } = useContext(RequestContext);
   const { content, metadata, promo } = articleData;
```

Comment:
Test coverage of this file has dropped - stating that it isn't covering `22, 23, 24, 26` by the snapshot tests. However, I can't see another way to cover these lines, since they have the requisite props being passed in https://github.com/bbc/simorgh/pull/1651/files#diff-7fe6e5e22f6ceb8837f76f59ca524a51R12